7-gce/gce/deployment_manager/bookshelf.jinja

```diff
@@ -0,0 +1,120 @@
+{#
+Copyright 2016 Google Inc. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+#}
+
+{% set NAME = "bookshelf-" + env["deployment"] %}
+{% set SERVICE = "bookshelf-" + env["deployment"] + "-frontend" %}
+
+
+resources:
+- name : {{ NAME }}
+  type: compute.v1.instanceTemplate
+  properties:
+    properties:
+      tags:
+        items:
+          - http-server
+      disks:
+        - boot: True
+          type: PERSISTENT
+          initializeParams:
+            sourceImage: {{ properties['machine-image'] }}
+            diskSizeGb: 10
+            diskType: pd-ssd
+      machineType: {{ properties['machine-type'] }}
+      serviceAccounts:
+          - email: default
+            scopes: {{ properties['scopes'] }}
+      metadata:
+        items:
+          - key: startup-script
+            value: |
+{% for line in imports['startup-script'].split('\n') %}
```

Comment:
ಠ_ಠ

You could try to use jinja's indent filter, maybe something like

``{{imports['startup-script']|indent(10)}}``